udata/templates/search.html

```diff
@@ -78,6 +80,7 @@
 {% endblock %}
 
 {% block toolbar_container %}
+{% if datasets %}
```

Comment:
OK, can be removed as dataset tab is removed too. But the tab activation should still be triggered